prow/plugins.yaml

```diff
@@ -18,7 +17,10 @@ triggers:
   - containerd/cri-containerd
   trusted_org: containerd
   join_org_url: "https://github.com/containerd/containerd/blob/master/MAINTAINERS"
-
+- repos:
+  - kubeflow/kubeflow
```

Comment:
Just use `kubeflow` to make this apply to the entire `kubeflow` Github org.

Reply:
Fantastic.